src/backend/web/local/bootstrap.py

```diff
@@ -201,5 +242,12 @@ def bootstrap_key(cls, key: str, apiv3_key: str) -> Optional[str]:
             for event_key in event_keys:
                 deferred.defer(cls.update_event, event_key, apiv3_key)
             return f"/events/{key}"
+        elif key.isalpha():
```

Comment:
Can we validate the district key name instead of allowing all letters?

Reply:
We do not, but we can add one if you want.